GetStream/stream-chat-android
# Add a way to disable showing notification after receiving a push

### 🎯 Goal

Add a way to disable showing notification after receiving a push

### 🛠 Implementation details

Added `NotificationConfig:: shouldShowNotificationOnPush` that returns `true` by default.

### 🧪 Testing

1. Check if push notification works as expected
2. Try to update lambda (e.g. return false) and check if everything works correctly

### ☑️ Checklist
